### PR TITLE
fix #1622

### LINKED ISSUE #1622: [FIXED] Bug with pvp zone
Seems PvP Zone is bugged at last TFS. When you attack a player at pvp zone you get pz locked (http://2.1m.yt/rO0BHKBl.png). Normally pvp zone doesn't get pz locked, only battle.

Any fix for this? Thanks.

### PR DESCRIPTION
**Pull request by vankk**
#1622

### CODE REVIEW

**Comment by sgioia9** [comment]
I don't like the way it's being fixed, but it's just my personal opinion. It looks forced.

**Comment by vankk** [comment]
We have another way to fix it? Also why looks forced to you, did not understand your view.

**Comment by sgioia9** [comment]
I misread, LGTM

**Comment by marksamman** [comment]
Thanks!

File: src/player.cpp [hunk 1]
```diff
@@ -3501,6 +3501,10 @@ void Player::onAttackedCreature(Creature* target)
 {
 	Creature::onAttackedCreature(target);
 
+	if (target && target->getZone() == ZONE_PVP) {
+		return;
+	}
+
```
Comment (WibbenZ) [inline src/player.cpp]: Trailing tab, remove it in another PR.

**Pull request merged**